packages/ember-metal/lib/mixin.js

```diff
@@ -590,7 +590,7 @@ Ember.anyUnprocessedMixins = false;
   @static
   @param arguments*
 */
-Mixin.create = function() {
+Mixin.create = () => {
   // ES6TODO: this relies on a global state?
   Ember.anyUnprocessedMixins = true;
   var M = this;
```

Comment:
`this` reference will be the the wrong scope, but I'm curious about the `ES6TODO` comment here.